zhaquirks/__init__.py

```diff
@@ -41,7 +41,7 @@ async def unbind(self):
         """Prevent unbind."""
         return (foundation.Status.SUCCESS,)
 
-    async def _configure_reporting(self, *args, **kwargs):
+    async def _configure_reporting(self, *args, **kwargs):  # pylint: disable=W0221
```

Comment:
Just curious, why not make the signature match?

Reply:
I was thinking about it, but particularly this implementation doesn't really care about the signature or parameters and in the future if the signature in zigpy has to change for any reason, it's just one less dependency to be a breaking change.
But could be a matching signature,